README.md

```diff
@@ -24,11 +24,13 @@ Project is automatically deployed to testing environment when pushing to develop
 
 ### Install events-helsinki-api-proxy locally
 
-Clone the repository (https://github.com/City-of-Helsinki/events-helsinki-api-proxy). Follow the instructions for running open-city-profile with docker.
+Clone the repository (https://github.com/City-of-Helsinki/events-helsinki-ui). Follow the instructions for running open-city-profile with docker.
```

Comment:
should be: https://github.com/City-of-Helsinki/events-helsinki-api-proxy

Reply:
yes I read this lazily and thought is copy-paste error: "Clone this repository" etc...

Reply:
fixed